reactor-core/src/test/java/reactor/core/scheduler/SchedulersMetricsTest.java

```diff
@@ -234,21 +247,20 @@ public void shouldReportExecutionTimes() {
 		}
 		phaser.arriveAndAwaitAdvance();
 
-		Timer timer = simpleMeterRegistry
+		Collection<Timer> timers = simpleMeterRegistry
```

Comment:
Why `Collection`? We filter by tag, isn't it unique enough?

Reply:
no, for a given SCHEDULER_ID there are several executors for the elastic and capped cases (parallel is historically initialized at 1 for that test)

Reply:
and if there are multiple matching meters, `.timer()` will return the first matching timer, which is not satisfactory